client/a8c-for-agencies/sections/marketplace/wpcom-overview/index.tsx

```diff
@@ -96,6 +118,70 @@ export default function WpcomOverview() {
 					quantity={ selectedCount.value }
 					discount={ selectedCount.discount }
 				/>
+
+				<HostingOverview
+					title={ translate( 'Powerful development & platform tools' ) }
+					subtitle={ translate( 'Build for developers, by developers' ) }
```

Comment:
Small typo: `Build` --> `Built`

Reply:
Good catch. I will fix this on the next PR. thanks